extensions/infinispan-client/deployment/src/test/java/org/quarkus/infinispan/client/deployment/InfinispanConfigurationSetupTest.java

```diff
@@ -44,6 +44,11 @@ public void infinispanConnectionConfiguration() {
         assertThat(configuration.security().ssl().provider()).isEqualTo("SSL_prov");
         assertThat(configuration.security().ssl().protocol()).isEqualTo("SSL_protocol");
         assertThat(configuration.security().ssl().ciphers()).containsExactlyInAnyOrder("SSL_cipher1", "SSL_cipher2");
+        assertThat(configuration.clusters().size()).isEqualTo(1);
+        assertThat(configuration.clusters().get(0).getClusterName()).isEqualTo("bsite");
+        assertThat(configuration.clusters().get(0).getCluster().get(0).host()).isEqualTo("bsite1");
+        assertThat(configuration.clusters().get(0).getCluster().get(0).port()).isEqualTo(32111);
+        assertThat(configuration.clusters().get(0).getClientIntelligence()).isEqualTo(ClientIntelligence.BASIC);
```

Comment:
Minor complaint: There are nicer ways to write this in AssertJ, but it's not a problem :)

Reply:
some refactored done